Sources/BrowserServicesKit/ContentBlocking/ContentBlockerRulesIdentifier.swift

```diff
@@ -23,12 +23,12 @@ public class ContentBlockerRulesIdentifier: Equatable, Codable {
     
     let name: String
     let tdsEtag: String
-    let tempListEtag: String
-    let allowListEtag: String
+    let tempListId: String
+    let allowListId: String
     let unprotectedSitesHash: String
     
     public var stringValue: String {
-        return name + tdsEtag + tempListEtag + unprotectedSitesHash
+        return name + tdsEtag + tempListId + allowListId + unprotectedSitesHash
```

Comment:
Allowlist was missing, as it used to be the same as tempListId (config etag) - it is not a case now so it has to be specified separately.